go/ocr2/decryptionplugin/decryption.go

```diff
@@ -82,6 +82,7 @@ func (dp *decryptionPlugin) Query(ctx context.Context, ts types.ReportTimestamp)
 	for _, request := range decryptionRequests {
 		ciphertext := &tdh2easy.Ciphertext{}
 		if err := ciphertext.UnmarshalVerify(request.Ciphertext, dp.publicKey); err != nil {
+			dp.decryptionQueue.SetResult(request.CiphertextId, nil, fmt.Errorf("cannot unmarshal the ciphertext: %w", err))
```

Comment:
Out of curiosity, why don't we catch incorrect ciphertexts with logging?

Reply:
The Functions product needs to provide users helpful error messages for debugging.  If a user provides invalid ciphertext, we need to be able to detect and inform them.

Reply:
I see. (Not really my business, but then I'd go with a new interface method like `ReportErr`, instead of overloading `SetResults`.)